Tools/Scripts/webkitpy/layout_tests/servers/web_platform_test_server.py

```diff
@@ -180,6 +185,9 @@ def _spawn_process(self):
         self._process = self._executive.popen(self._start_cmd, cwd=self._doc_root_path, shell=False, stdin=self._executive.PIPE, stdout=self._wsout, stderr=self._wsout)
         self._filesystem.write_text_file(self._pid_file, str(self._process.pid))
 
+        if self._dns_server:
+            self._dns_server.start_thread()
```

Comment:
I think @JonWBedard's comment above is probably more applicable here, and specifically whether we want this to be a subprocess or a thread.

The main downside to this being a thread is the fact that while the DNS server is responding its thread needs to hold the [GIL](https://docs.python.org/3/glossary.html#term-global-interpreter-lock), and potentially will have had wait to acquire the GIL.

That said, if this didn't have any notable impact to EWS runtime then it's probably fine.